test/integration/redis_proxy_integration_test.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "test/integration/integration.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace {
+class RedisProxyIntegrationTest : public testing::TestWithParam<Network::Address::IpVersion>,
```

Comment:
nit, newer tests just do away with the header file. Can you merge this into the cc file?

Reply:
done